FAQ.md

```diff
@@ -104,10 +104,10 @@ dependency, you have a couple of options:
 2. Use an override.
 
 Overrides are a sledgehammer, and should only be used as a last resort. While
-dependencies and overrides are declared in the same way in `Gopkg.toml`, they
+constraints and overrides are declared in the same way in `Gopkg.toml`, they
 behave differently:
 
-* Dependencies:
+* Constraints:
```

Comment:
I thought of replacing "Dependencies" with "Constraints" around this section, but resisted thinking we might keep using "Dependencies" for direct dependencies as a reference to `constraint`s.
👍